src/govuk/components/error-summary/error-summary.test.js

```diff
@@ -13,6 +13,15 @@ describe('Error Summary', () => {
     expect(moduleName).toBe('govuk-error-summary')
   })
 
+  it('removes the tabindex attribute on blur', async () => {
```

Comment:
Do we need to check that the `tabindex` is being set correctly too? I think this test could pass if `tabindex` was never set in the first place?

Reply:
Good question. This test does pass, but the test before it fails – because tabindex isn't set, focus doesn't move to the error summary. So it's sort of there but implicit at the moment. Happy to add an explicit test for it though.

Reply:
@36degrees I think it'd be good to have an explicit test so we don't unintentionally change some behaviour/test without realising 🙂